warehouse/templates/pages/classifiers.html

```diff
@@ -22,7 +22,7 @@ <h1 class="page-title">{% trans %}Classifiers{% endtrans %}</h1>
     <p>{% trans %}Each project's maintainers provide PyPI with a list of "Trove classifiers" to categorize each release, describing who it's for, what systems it can run on, and how mature it is.{% endtrans %}</p>
     <p>{% trans %}These standardized classifiers can then be used by community members to find projects based on their desired criteria.{% endtrans %}</p>
     <p>
-      {% trans ppug_href='https://packaging.python.org/tutorials/distributing-packages/#configuring-metadata', pep301_href='https://www.python.org/dev/peps/pep-0301/#distutils-trove-classification', title=gettext('External link') %}
```

Comment:
What happened to the pep301 link? Was it no longer needed? I see a call for it a couple of lines lower.

Reply:
Bleh, bad substitution. Fixing now.